language-support/hs/bindings/test/DA/Ledger/Sandbox.hs

```diff
@@ -40,12 +41,15 @@ sandboxProcess SandboxSpec{dar} = do
 startSandboxProcess :: SandboxSpec -> IO (ProcessHandle,Maybe Handle)
 startSandboxProcess spec = do
     processRecord <- sandboxProcess spec
-    (_,hOutOpt,_,proh) <-
+    (_,hOutOpt,Just hErrOpt,proh) <-
         createProcess processRecord {
         std_out = CreatePipe,
         std_err = CreatePipe, -- Question: ought the pipe to be drained?
         create_group = True  -- To avoid sending INT to ourself
         }
+    _ <- forkIO $ forever $ do
+        line <- hGetLine hErrOpt
+        print line
```

Comment:
This is odd. print and not putStrLn? Sending to stdout and not stderr?

Reply:
oops that was only for debugging